packages/jest-reporters/src/__tests__/GitHubActionsReporter.test.ts

```diff
@@ -5,20 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {afterEach, describe, expect, jest, test} from '@jest/globals';
```

Comment:
No need to import. Something did not work?

```suggestion
```

Reply:
Yeah. Apparently there was an issue with my local repo that did not allow typescript to recognize the globals. Fixed now